### PR TITLE
Wk/taco 35 zipkin server integration test

### PR DESCRIPTION
**Pull request by manimaul**
added test to verify span is sent

### CODE REVIEW

File: xio-core/src/test/java/com/xjeffrose/xio/fixtures/OkHttpUnsafe.java [hunk 1]
```diff
@@ -32,15 +32,6 @@
     return keyManagerFactory.getKeyManagers();
   }
 
-  public static KeyManager[] getEmptyKeyManager() throws Exception {
```
Comment (manimaul) [inline xio-core/src/test/java/com/xjeffrose/xio/fixtures/OkHttpUnsafe.java]: this function is unused and a duplicate

Comment (pdex) [reply xio-core/src/test/java/com/xjeffrose/xio/fixtures/OkHttpUnsafe.java]: This whole class can be removed and the one in xio-test can be used instead.

File: xio-core/src/main/java/com/xjeffrose/xio/http/StreamingRequestData.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
   public StreamingRequestData(Request request, StreamingData data, TraceInfo traceInfo) {
     this.request = request;
     this.data = data;
-    this.traceInfo = traceInfo;
+    this.traceInfo = traceInfo == null ? new TraceInfo(request.headers()) : traceInfo;
```
Comment (pdex) [inline xio-core/src/main/java/com/xjeffrose/xio/http/StreamingRequestData.java]: 👍 nice ternary operator

File: xio-core/src/main/java/com/xjeffrose/xio/tracing/XioTracing.java [hunk 3]
```diff
@@ -14,13 +14,13 @@
 
 public class XioTracing {
 
-  private final Tracing tracing;
+  protected final Tracing tracing;
```
Comment (pdex) [inline xio-core/src/main/java/com/xjeffrose/xio/tracing/XioTracing.java]: Can we replace this with a @VisibleForTesting getter?

**Pull request merged**